auth/create-auth-tables.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE `user_data` (
+  `id` INT(11) NOT NULL AUTO_INCREMENT,
+  `username` varchar(255) NOT NULL,
+  `user_id` varchar(255) NOT NULL,
+  `email` varchar(255) DEFAULT NULL,
+  `developer` tinyint(1) DEFAULT NULL,
+  `service_account` tinyint(1) DEFAULT NULL,
+  `namespace_name` varchar(255) DEFAULT NULL,
+  `gsa_email` varchar(255) NOT NULL,
+  `ksa_name` varchar(255) DEFAULT NULL,
+  `bucket_name` varchar(255) NOT NULL,
+  `gsa_key_secret_name` varchar(255) NOT NULL,
+  `jwt_secret_name` varchar(255) NOT NULL,
+  `sql_user_secret` varchar(255) DEFAULT NULL,
+  `sql_admin_secret` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `user_id` (`user_id`),
+  KEY `email` (`email`),
+  KEY `username` (`username`)
+) ENGINE=InnoDB;
+
+CREATE TABLE `sessions` (
```

Comment:
I think this could potentially be improved with a trigger that automatically deletes itself on expiration (or in the future, takes another action, like setting an expired field). I may not want to let a future developer accidentally provide an expired session to a user (say by forgetting to check max_age).

Reply:
I couldn't see how to create a a trigger here, but I added an event every 5 minutes to delete expired sessions.

Reply:
Looks great!